src/App.jsx

```diff
@@ -5,14 +5,15 @@ import "./App.css";
 import { getAllArticles } from "./utils";
 import ArticleContainer from "./components/ArticleContainer";
 import NavBar from "./components/NavBar";
+import ArticleCard from "./components/ArticleCard";
 
 function App() {
   return (
     <>
     <NavBar/>
       <Routes>
         <Route path="/" element={<ArticleContainer />} />
-        
+        <Route path="/article/:article_id" element={<ArticleCard/>} />
```

Comment:
This route should probably be `articles` (plural) `/ :article_id` to be conventional